cmake/CodeStyle.cmake

```diff
@@ -20,22 +20,51 @@
 cmake_minimum_required(VERSION 3.1)
 
 option(APPLY_CPP_CODE_STYLE "Apply C++ code style using clang format" OFF)
+option(CHECK_CPP_CODE_STYLE "Check C++ code style using clang format" ON)
 
 find_program(CLANG_FORMAT clang-format DOC "Clang format executable")
 
+# Python
```

Comment:
Can the Python check be conditional on the code style check being enabled?

Reply:
yes, the check can be conditional, it is updated in the commit [cedbfcb](https://github.com/LunarG/gfxreconstruct/pull/258/commits/cedbfcba02d9e340e0d7c1a8e4d128497ab353cc)